fbw-a32nx/src/systems/instruments/src/EFB/Store/features/globalSettings.ts

```diff
@@ -0,0 +1,33 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+// Store for all settings which should be global for the EFB but not neccesary for the simvars.
+
+interface GlobalSettingsState {
+    cabinAnnouncementsActive:boolean,
+    passengerAmbienceActive:boolean,
+    boardingMusicActive:boolean
+}
+
+let initialState:GlobalSettingsState = {cabinAnnouncementsActive: true, passengerAmbienceActive: true, boardingMusicActive: true}
```

Comment:
Is there any reason you didn't want to make this a `const`?

The only reason you really would is if you need to mutate it further in its life, but I can't seen that happening anytime soon!

Reply:
Yes, const is the right type. I´ll add it in my next commit todays evening.